Source/Plugins/SyntheticBuilding/SyntheticBuildingFilters/InsertPrecipitatePhases.h

```diff
@@ -370,7 +370,7 @@ class SyntheticBuilding_EXPORT InsertPrecipitatePhases : public AbstractFilter
    * @param add Determines amount to iterate RDF bins
    * @param double_count Determines whether to double count items in bins
    */
-  void determine_currentRDF(int32_t featureNum, int32_t add, bool double_count);
+  void determine_currentRDF(int32_t gnum, int32_t add, bool double_count);
```

Comment:
What is gnum?  featureNum is a lot more consistent and seems to be correct based on the description.